aikau/src/main/resources/alfresco/forms/TabbedControls.js

```diff
@@ -83,9 +83,9 @@ define(["alfresco/layout/AlfTabContainer",
         "dojo/_base/declare",
         "dojo/_base/lang",
         "dojo/_base/array"], 
-        function(HorizontalWidgets, LayoutMixin, declare, lang, array) {
+        function(AlfTabContainer, LayoutMixin, declare, lang, array) {
```

Comment:
It's not changing the import, just the variable name... anyone extending would have to declare their own variable names so it shouldn't impact them